representationtests.sh

```diff
@@ -0,0 +1,27 @@
+#
+# Lincheck
+#
+# Copyright (C) 2019 - 2025 JetBrains s.r.o.
+#
+# This Source Code Form is subject to the terms of the
+# Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+# with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+# jdk 11 is SLOW! So remove it if you don't expect files to change
+jdks=("8" "11" "13" "15" "17")
```

Comment:
Why there are no 19, 20 and 21 versions? 
We also test on these versions on CI.